test/Microsoft.Azure.ServiceBus.UnitTests/BrokeredMessageTests.cs

```diff
@@ -12,9 +12,11 @@ public class BrokeredMessageTests
         [Fact]
         async Task BrokeredMessageOperationsTest()
         {
+            var messagingFactory = new MessagingFactory();
+
             // Create QueueClient with ReceiveDelete,
             // Send and Receive a message, Try to Complete/Abandon/Defer/DeadLetter should throw InvalidOperationException()
-            var queueClient = QueueClient.CreateFromConnectionString(
+            var queueClient = (QueueClient)messagingFactory.CreateQueueClientFromConnectionString(
```

Comment:
I agree. All the tests should be converted to the interface equivalents instead of this hack.
Thanks for the PR #85